homeassistant/components/proxmoxve/__init__.py

```diff
@@ -306,7 +325,7 @@ def __init__(self, host, port, user, realm, password, verify_ssl):
         self._proxmox = None
         self._connection_start_time = None
```

Comment:
`_connection_start_time` is fully unused... it looks like a leftover from a previous implementation.
```suggestion
```